src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/Formatting/FormattingHandler.ts

```diff
@@ -38,6 +49,12 @@ export class FormattingHandler {
                 return this.emptyFormattingResponse;
             }
 
+            const textDocument = await vscode.workspace.openTextDocument(razorDocumentUri);
```

Comment:
Also, is this line needed? I think `documentManager.getDocument` already open the document for us

Reply:
I'm not actually needing the document opening part of this, but this was the only API I could find that gave me a `TextDocument` I needed to call the document synchronizer. If there is another/better way to do this, I'm all ears. I copied this from another PR I had, which I'm unable to link to, so could be a few spots to update if there is a better one.